src/web-fls-quiz/Services/MailService.cs

```diff
@@ -28,6 +28,21 @@ public class MailService : IMailService
             _contextAccessor = contextAccessor;
         }
 
+        public async Task<string> GetAdminEmail()
+        {
+            var result = await _configurationService.GetAdminEmail();
+
+            if (!string.IsNullOrEmpty(result))
+                return result;
+
+            result = await _configurationService.GetAdminEmailUsingNotConfirmedConfiguration();
```

Comment:
We should just leave configuring it in Vault only. If it is not configured there, we assume Vault is just as good as dead to us.